test/integration/release-action-integration.ts

```diff
@@ -1,46 +1,87 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
 import { expect } from 'chai'
-import { before, describe, test, after } from 'mocha'
+import { before, describe, test } from 'mocha'
 import { Environment } from '../../lib/entities/environment'
-import { Release } from '../../lib/entities/release'
+import { Release, ReleasePayload } from '../../lib/entities/release'
+import { ReleaseAction } from '../../lib/entities/release-action'
 import { Space } from '../../lib/entities/space'
 import { TestDefaults } from '../defaults'
-import { getDefaultSpace } from '../helpers'
+import { createTestSpace, initClient, initPlainClient } from '../helpers'
 import { makeLink } from '../utils'
 
-describe('ReleaseAction Api', async function () {
+describe('ReleaseAction Api', async () => {
   let testSpace: Space
   let testEnvironment: Environment
   let testRelease: Release
+  let testRelease2: Release
+
+  let testReleaseAction: ReleaseAction
+  let testReleaseAction2: ReleaseAction
 
   before(async () => {
-    testSpace = await getDefaultSpace()
+    testSpace = (await createTestSpace(initClient(), 'Release Actions')) as unknown as Space
```

Comment:
this will actually spin up the new space, are you aware of that?

Reply:
The team responsible for the project has setup an `autoClean` feature whenever we use `createTestSpace` -- there's also no space limits for the org these tests run on. They are cleaned up after a certain "age" that the space reaches.

It's a pretty good approach since now each test runs in isolation from entities created in other tests.

Reply:
I also refactored `releases-integration` to use the same approach in a previous PR.